bin/setenv-docker-customize.sh

```diff
@@ -556,6 +538,42 @@ else
   # Time after which a token will be invalid. The use will then be considered offline.
   add_in_chat_configuration "chatTokenValidity=30000"
 
+  if [ "${EXO_CHAT_SERVER_STANDALONE}" = "false" ]; then
+    # Mongodb configuration (for the Chat)
+    add_in_chat_configuration "# eXo Chat mongodb configuration"
+    add_in_chat_configuration "dbServerHost=${EXO_MONGO_HOST}"
+    add_in_chat_configuration "dbServerPort=${EXO_MONGO_PORT}"
+    add_in_chat_configuration "dbName=${EXO_MONGO_DB_NAME}"
+    if [ "${EXO_MONGO_USERNAME}" = "-" ]; then
+      add_in_chat_configuration "dbAuthentication=false"
+      add_in_chat_configuration "#dbUser="
+      add_in_chat_configuration "#dbPassword="
+    else
+      add_in_chat_configuration "dbAuthentication=true"
+      add_in_chat_configuration "dbUser=${EXO_MONGO_USERNAME}"
+      add_in_chat_configuration "dbPassword=${EXO_MONGO_PASSWORD}"
+    fi
+
+    # The notifications are cleaned up every one hour by default.
+    add_in_chat_configuration "chatCronNotifCleanup=0 0/60 * * * ?"
+    # When a user reads a chat, the application displays messages of some days in the past.
+    add_in_chat_configuration "chatReadDays=30"
+
+  else
+
+    ### Uninstall chat addon
+    echo "INFO: External chat server configured, removing charServer.war"
+    rm -vf ${EXO_APP_DIR}/webapps/chatServer.war
```

Comment:
this is not needed as we remove exo-chat add-on few lines after

Reply:
lines removed